packages/@ember/-internals/glimmer/lib/helpers/each-in.ts

```diff
@@ -87,19 +96,23 @@ import { Opaque } from '@glimmer/util';
 
   For example, given a `user` object that looks like:
 
-  ```javascript
-  {
-    "name": "Shelly Sails",
-    "age": 42
+  ```app/components/developer-details.js
+  import Component from '@glimmer/component';
+
+  export default class extends Component {
+    developer = {
```

Comment:
Should this be tracked? Does the example require updates?

Reply:
Make it tracked for consistency.